docs/adguard-browser-extension/mv3-version.md

```diff
@@ -0,0 +1,159 @@
+---
+title: AdGuard Browser Extension for Chrome MV3
+sidebar_position: 5
+---
+
+As MV2 extensions are being phased out in the Chrome Web Store, Chrome users may need to switch to MV3 extensions that are compatible with Chrome’s new API Manifest V3. Our thoughts on this topic are extensively discussed on [our blog](https://adguard.com/en/blog/tag/manifest-v3.html).
+
+Our MV3 extension effectively blocks ads and trackers while seamlessly managing social widgets, banners, and video ads. Most users will not notice any operational differences, but there are some limitations and changes to be aware of.
+
+[Where to find our extensions](#where-to-find-our-extensions)
+
+[Main changes in features and settings](#main-changes-in-features-and-settings)
+
+[Limitations](#limitations)
```

Comment:
what is it for?

Reply:
I guess it is sort of manually created TOC. but is it really needed if there is an automatically generated one on the side?
https://uploads.adguard.org/slQLvskoMV9DUtaE2a.png

Reply:
this manually created ToC can be omitted if you think it's unnecessary